### PR TITLE
Update kubernetes installation docs

### PR DESCRIPTION
**Pull request by otherjamesbrown**
I have brought the Kubernetes install documentation inline with the newer style of install docs.  I have also removed the option to install onto a remote ETCD cluster as that is not the standard install path

### CODE REVIEW

File: docs/install/kubernetes.md [hunk 1]
```diff
@@ -3,149 +3,143 @@ title: "Kubernetes"
...
+1. The first set of commands adds the Ondat Helm repository and ensures a updated local cache.
 
-    ```bash
-    kubectl get all --namespace=storageos
-    kubectl get all --namespace=storageos-etcd
-    kubectl get storageclasses | grep "storageos"
-    ```
+```bash
+helm repo add ondat https://ondat.github.io/charts && \
+helm repo update && \
```
Comment (Arau) [inline docs/install/kubernetes.md:78]: This line is unfinished with the `\`. I guess the

```
&& \
```
Need to be removed

File: docs/install/kubernetes.md [hunk 2]
```diff
@@ -3,149 +3,143 @@ title: "Kubernetes"
...
+  --set ondat-operator.cluster.portalManager.enabled=true \
+  --set ondat-operator.cluster.portalManager.clientId=37540b25-285c-4326-b76c-742100723ac3 \
+  --set ondat-operator.cluster.portalManager.secret=e946f84a-e6c0-4afd-9087-f9cdd6906aa5 \
+  --set ondat-operator.cluster.portalManager.apiUrl=https://portal-setup-7dy4neexbq-ew.a.run.app \
+  --set ondat-operator.cluster.portalManager.tenantId=16e51eb9-37cf-4103-9a31-9e2cdaeec373 \
+  --set etcd-cluster-operator.cluster.replicas=3 \
+  --set etcd-cluster-operator.cluster.storage=6Gi \
+  --set etcd-cluster-operator.cluster.resources.requests.cpu=100m \
+  --set etcd-cluster-operator.cluster.resources.requests.memory=300Mi
+```
```
Comment (Arau) [inline docs/install/kubernetes.md:96]: People very often copy&paste the cli commands without paying much attention. At the end of point 2, I would add a comment indicating that the portalManager.{clientId,secret,apiUrl,tenantId} need to be filled with their details. 
I would suggest with a warning comment. 

A different option is to put ENV vars at the beginning of the sectiong, before helm install with the details, so it is very visible that need to be filled

```
PORTAL_CLIENTID=37540b25-285c-4326-b76c-742100723ac3
PORTAL_SECRET=e946f84a-e6c0-4afd-9087-f9cdd6906aa5 \
...
helm install ondat ...
```

File: docs/install/kubernetes.md [hunk 3]
```diff
@@ -3,149 +3,143 @@ title: "Kubernetes"
...
+helm repo add ondat https://ondat.github.io/charts && \
+helm repo update && \
+```
 
-### Option B - Using An External `etcd` Deployment
+2. The last command installs Ondat with a set of basic install parameters that are sufficient for a basic trial installation.
 
-#### Step 1 - Setup An `etcd` Cluster
+```bash
+helm install ondat ondat/ondat \
```
Comment (Arau) [inline docs/install/kubernetes.md:84]: We are missing the etcd storage class, or a mention that a default StorageClass needs to exist.

File: docs/install/kubernetes.md [hunk 4]
```diff
@@ -3,149 +3,143 @@ title: "Kubernetes"
...
-    ```
+![kubectl-storageos Install](/images/docs/install/PluginInstall.png)
 
-#### Step 3 - Installing Ondat
+This command uses the `kubectl-storageos` plugin command with a set of basic install parameters that are sufficient for a basic trial installation. The installation process may take a few minutes.
 
-1. Define and export the `STORAGEOS_USERNAME` and `STORAGEOS_PASSWORD` environment variables that will be used to manage your Ondat instance. In addition, define and export a `ETCD_ENDPOINTS` environment variable, where the value will be a list of `etcd` endpoints as comma-separated values noted down earlier in **Step 2**.
+```bash
+kubectl storageos install \
+  --include-etcd=true \
```
Comment (Arau) [inline docs/install/kubernetes.md:110]: We are missing the etcd storage class, or a mention that a default StorageClass needs to exist.

File: docs/install/kubernetes.md (hunk 1 above)
Comment (otherjamesbrown) [reply docs/install/kubernetes.md:78]: The command is not meant to be copied from the documentation, it is explaining the command that the portal gives them.  To stop people copying/pasting it I will change it to a clipped screenshot and highlight that it should be copied from the portal

File: docs/install/kubernetes.md (hunk 2 above)
Comment (otherjamesbrown) [reply docs/install/kubernetes.md:96]: The command is not meant to be copied from the documentation, it is explaining the command that the portal gives them.  To stop people copying/pasting it I will change it to a clipped screenshot and highlight that it should be copied from the portal

File: docs/install/kubernetes.md (hunk 3 above)
Comment (otherjamesbrown) [reply docs/install/kubernetes.md:84]: The installation of the StorageClass is added to Prerequisites Section 3 - Installing a Local Path Provisioner

File: docs/install/kubernetes.md (hunk 4 above)
Comment (otherjamesbrown) [reply docs/install/kubernetes.md:110]: The installation of the StorageClass is added to Prerequisites Section 3 - Installing a Local Path Provisioner

**Review by Arau — APPROVED** [review]

**Review by Arau — DISMISSED** [review]

**Review by Arau — DISMISSED** [review]

**Comment by Arau** [comment]
I'm wondering if the Quick install should be at the top of the list. If you agree, I'll put it high on the list and commit here.

**Review by Arau — APPROVED** [review]

**Comment by Arau** [comment]
Fixing conflicts

**Review by Arau — APPROVED** [review]

**Pull request merged**